src/VisualStudio/Xaml/Impl/Features/Diagnostics/XamlDiagnostic.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.VisualStudio.LanguageServices.Xaml.Features.Diagnostics
+{
+    public class XamlDiagnostic
```

Comment:
where is this type used? i couldn't really see it anywhere afaict.

Reply:
It should be internal.

---
In reply to: [516286634](https://github.com/dotnet/roslyn/pull/49145#discussion_r516286634) [](ancestors = 516286634)